service/models.py

```diff
@@ -67,10 +67,15 @@ def create(self):
         db.session.add(self)
         db.session.commit()
 
-    def update(self):
+    def update(self, new_data):
         """
         Updates a Inventory to the database
         """
+        self.name = new_data.name or self.name
```

Comment:
so if a new_data value is present it will take new_data and not the existing name?

Reply:
That's right. It first checks `new_data.name`. If it is not `None`, it is assigned. Else, we take the value from `self.name`.